docs/tutorials/porting.md

```diff
@@ -0,0 +1,225 @@
+## Mbed OS porting guide
```

Comment:
I don't see any of the images on this page. Do we want to include them here or delete them?

Reply:
The last commit replaced all of them with the single tree structure at the bottom of the document.